returnn/compile.py

```diff
@@ -228,12 +232,16 @@ def __init__(
         :param returnn_config: RETURNN config object
         :param checkpoint: Path to the checkpoint for export
         :param device: target device for graph creation
+        :param input_names: sequence of model output names
```

Comment:
```suggestion
        :param input_names: sequence of model input names
```

Reply:
Actually, also should explain what happens if you don't pass it.
```suggestion
        :param input_names: sequence of model input names.
            If not specified, will automatically determine from extern_data when available in returnn_config.config.
```